pkgs/applications/radio/rbfeeder/default.nix

```
@@ -0,0 +1,71 @@
+{ lib, stdenv
+, fetchFromGitHub
+, pkg-config
+, curl
+, glib
+, jansson
+, ncurses
+, protobuf
+, protobufc
+, hackrf
+, libbladeRF
+, libusb1
+, rtl-sdr
+, limesuite
+, DarwinTools
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rbfeeder";
+  version = "1.0.8";
+
+  src = fetchFromGitHub {
+    owner = "airnavsystems";
+    repo = pname;
+    rev = "17a5f65da1b9ca1a01c636eb728ea6db9010b708";
+    sha256 = "sha256:16s1ybz2r0fn92rdpwhlkcys0sgwjsfbqmkdccp7n2k7vzsi5y4j";
```

Use SRI format (`nix hash to-sri <sha256:hash>`):

```suggestion
    sha256 = "sha256-kvgS9d9nCnsuY21WvJyW/GmgPZsU8tuySNaBLP7yQZs=";
```

---

Thanks. fixing.